lightning/src/routing/router.rs

```diff
@@ -575,7 +591,7 @@ where L::Target: Logger {
 	//    any ~sufficient (described later) value.
 	//    If succeed, remember which channels were used and how much liquidity they have available,
 	//    so that future paths don't rely on the same liquidity.
-	// 3. Prooceed to the next step if:
```

Comment:
If it's not much of a problem, could you keep the initial clean-up commit separate rather than squashing it with the others? May be able to recover it with `git reflog`.

Reply:
Sure thing, I now split of https://github.com/lightningdevkit/rust-lightning/pull/1234/commits/689518520e9daadb7fddd2b836bc19815ef24921.